docs/dev/framework/templates/_index.md

```diff
@@ -164,7 +197,7 @@ but somewhere else entirely, you could extend that template, override the `headl
 block with empty content and instead show the headline in a custom location within
 the `content` block.
 
-```php
+```html
```

Comment:
_Note:_ this is because the syntax highlighter cannot deal with PHP code that contains multiple occurrences of `<?php … ?>`. Rendering it as `html` looks much better.